.changeset/gentle-sloths-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/next': patch
```

Comment:
Not blocking, but this is probably doesn't require even a patch version bump

Reply:
That isn't a thing I've ever heard of! I've always seen them required in my time here but running changeset with `--empty` will generate an empty one.

Reply:
Or just manually delete the content between the `---`s

Reply:
You can push an empty changeset to make the bot happy, but anyways, the changeset is not a required check, so it's not a requirement.